scm-webapp/src/main/java/sonia/scm/security/BearerRealm.java

```diff
@@ -92,6 +93,11 @@ protected AuthenticationInfo doGetAuthenticationInfo(AuthenticationToken token)
     checkArgument(token instanceof BearerToken, "%s is required", BearerToken.class);
 
     BearerToken bt = (BearerToken) token;
+
+    if (isJwtTokenExpired(((BearerToken) token).getCredentials())) {
```

Comment:
Why do we need to parse the token here? Where do we now that this is a JWT token? The BearerToken could be a Opaque Token or an encrypted JWT. The jwt expiration logic belongs to the `JwtAccessTokenResolver`.